resources/definitions/fdmprinter.def.json

```diff
@@ -3231,7 +3225,7 @@
                     "description": "Extrusion width based correction factor on the equalized flow. The target flow (in mm³/s) is reduced by the percentual difference in line width multiplied by this factor. A factor above zero means that thin lines are adjusted to be even faster than simple flow equalization would speed them up to. Conversely, wider lines are slowed down more. A positive value can help to compensate for the width depended back pressure the previous layer exterts on the line.",
                     "type": "float",
                     "unit": "mm³/s",
-                    "default_value": 0.0,
```

Comment:
I don't think we should be turning on equalization for every printer. Printers with Linear advance shouldn't need flow equalization. Although how they would achieve variable width is an open question.

Reply:
@BagelOrb FYI your suggestion about using percentage (ratio) instead of mm**3/s is implemented in the Arachne branches.